Aptoide/appcoins-wallet-android
# [APPC_1077] Implement Focus on fiat

**NOTE:** Do not accept this PR before there’s a service to convert from ETH to fiat.

**What does this PR do?**

Changes the balance on the transaction view, so that it shows the 3 currencies if the user has more than 0.01 cents in that currency and if that currency is converted to fiat (0,0001 in ether’s case).
Adds a skeleton animation on those values while the values are loading
Adds the navigation to the third BN item
Switches positions of fiat  and appc value in the transaction screens

**Where should the reviewer start?**

TransactionsActivity.java

**How should this be manually tested?**

  Check if the payment screens match the ones in zeplin
  Check if the values follow the rules described on the ticket and check something breaks

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/APPC-1077
